### DISCUSSION TITLE
Welcome to Meshtastic Discussions!

### DISCUSSION THREAD

**Discussion by rcarteraz** [meshtastic/Discussions#1] [Announcements]
Hey everyone!

We’re excited to officially welcome you to Meshtastic Discussions on GitHub! 🎉 Moving here from Discourse allows us to create a more centralized and integrated space for the community to interact with the contributors and developers working on Meshtastic.

Why the Move?

- **Closer Interaction with Developers:** Since we’re already active on GitHub, this move makes it easier for us to engage quickly with your questions, ideas, and feedback. You’ll have more direct access to the developers and contributors responsible for Meshtastic.
- **Better Integration with the Development Process:** GitHub Discussions seamlessly connects with our repositories. When issues are identified, we can easily convert topics to issues and transfer them to the relevant repo for tracking and resolution. This will help streamline development and make sure nothing falls through the cracks.
- **Simplified Community Experience:** Consolidating platforms means fewer places to check, so it’s easier for all of us to stay engaged. We’ve noticed the community has been more active on other platforms, and GitHub Discussions aligns well with our focus on continuous improvement.

Please take a moment to review our [Code of Conduct](https://meshtastic.org/docs/legal/conduct/) to help keep this space respectful and productive. We’re excited to hear your thoughts, share updates, and collaborate with you all more closely. Let’s make Meshtastic even better together!

Thank you for being part of our journey, and welcome to GitHub Discussions!

**Comment by n0qbh** [comment]
Are the conversations here starting from scratch or are you moving the discourse discussions here?

**Reply by rcarteraz** [reply]
Hi -- We will not be importing the conservations from Discourse. Feel free to reference a link to continue a conversation as Discourse is still accessible in read-only.
